src/styles/index.css

```diff
@@ -0,0 +1,237 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.header__item {
+  margin-left: 20px;
+}
+
+.header__item:first-child {
+  margin-left: 0;
+}
+
+.header__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+  transition-duration: transition-duration;
```

Comment:
The `transition-duration` should be set to `300ms` instead of `transition-duration` to correctly apply the transition duration as per the task requirements.